libs/platform/src/lib/utils/async-strategy/function-strategy.class.ts

```diff
@@ -0,0 +1,12 @@
+import { isFunction } from '../lang';
+import { SubscriptionStrategy } from './subscription-strategy.interface';
+
+/**
+ * @description Executes function and passes returned value into callback function.
+ */
+export class FunctionStrategy implements SubscriptionStrategy {
+    createSubscription(fn: Function, updateLatestValue: (v: any) => any): any {
```

Comment:
according to the interface it must return promise

Reply:
Interface changed. Now it returns Promise or void